pkgs/by-name/ap/aprx/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, perl
+}:
+
+stdenv.mkDerivation {
+  pname = "aprx";
+  version = "2.9.1-unstable-2021-09-21";
+
+  src = fetchFromGitHub {
+    owner = "PhirePhly";
+    repo = "aprx";
+    rev = "2c84448fe6d897980234961a87ee4c1d4fad69ec";
+    sha256 = "sha256-01PB7FaG8GmPm1U15/3g1CfQwdYmf3ThZFdVh2zUAl4=";
+  };
+
+  buildInputs = [ perl ];
```

Comment:
If perl is needed for a pre-processor during the build, for example, it should be in nativeBuildInputs. Perl does not provide any header files that I'm aware of, so I think that you need to change this to nativeBuildInputs.

Reply:
Ahh good catch, let me move it to `nativeBuildInputs`

Reply:
Done